scripts/train_contract_model.py

```
@@ -368,7 +377,7 @@ def run_quality_gate(
     if baseline_metrics_json.exists():
         cmd.extend(["--baseline-metrics-json", str(baseline_metrics_json)])
 
-    subprocess.run(cmd, check=True)
+    subprocess.run(cmd, check=True, capture_output=True, text=True)
```

Ruff (0.15.9)

[error] `subprocess` call: check for execution of untrusted input

(S603)